CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 Mapbox welcomes participation and contributions from everyone.  If you'd like to do so please see the [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) first to get started.
 
+## 8.5.0-alpha.3 - October 18, 2019
+
+### Bug fixes
+ - Fixed runtime exceptions that occurred when a manually built camera object without padding was serialized [#15788](https://github.com/mapbox/mapbox-gl-native/pull/15788)
+ - Fixed icon-text-fit to take in account the additional 1px texture padding when scaling [#15634](https://github.com/mapbox/mapbox-gl-native/pull/15634)
+ - Fixing occasional crashes during downloading big number (thousands) of offline regions [#15818](https://github.com/mapbox/mapbox-gl-native/pull/15818)
```

Comment:
I think @chloekraw have a few comments at https://github.com/mapbox/mapbox-gl-native/pull/15824#discussion_r336337385

> nit: start entries with the present tense verb (Fix) instead of the gerund
>I don’t think “big number” is necessary - “thousands” would suffice
>Suggestion:
> >Fix occasional crashes when downloading thousands of offline regions

Reply:
nice catch! @pengdev